tests/universal/keyexpr.cxx

```diff
@@ -83,40 +82,71 @@ void canonize() {
 }
 
 void concat() {
+#ifdef ZENOHCXX_ZENOHC
```

Comment:
These are not support in Zenoh-Cpp over Zenoh-Pico?
Maybe we need to add `concat` in Zenoh-Pico

Reply:
I think yes. The function `z_keyexpr_concat` and `z_keyexpr_join` looks like not zenoh-c specific, they should be in zenoh-pico too